src/sentry/ratelimits/cardinality.py

```diff
@@ -303,11 +305,17 @@ def check_within_quotas(
         for request in requests:
             granted_hashes = []
 
+            set_count = sum(set_counts[k] for k in self._get_read_sets_keys(request, timestamp))
+
+            metrics.timing(
```

Comment:
```suggestion
            metrics.incr(
```

Reply:
`timing` is a (poorly named) distribution metric, `incr` is a counter metric. we are interested in the min/max/avg of the `set_count` value, if we `incr` then we can basically only access `sum`.

`incr` would produce graphs that change based on how often rate limits are checked, but we don't really want that